Collaboration.md

```diff
@@ -0,0 +1,155 @@
+
+## #3 Redesign
+
+An existing project is maturing and wants to improve the overall design, which may also means finding a designer and adopting design processes.
+
+The term redesign is often used too quickly. Many times, it is better to systematically improve the existing design than start from scratch. This should be carefully considered.
+
+Steps we can take:
+
+1. Identify goals and vision
+1. Examine the current state of the project
+1. Compare vision to current state
+1. Break down work-to-be-done into manageable parts
+	- Example:
+		1. Refine brand
+		1. User Interface audit
+		1. Create a UI style guide
+		1. Implement UI style guide (without touching interaction design)
+		1. Revise individual application screens and user flows one-by-one
+1. Prioritize
+	- Consider whether to start with the application or the website
+	- Consider level of effort for each change
+	- Consider availability of contributors
+	- Consider time to adopt new workflows
+1. Plan collaboration
+	- Good idea to start with a design review to present the project and create initial interest
+
+Designers who want to tackle redesigns should ensure they have the availability to regularly contribute for longer periods of time. How much time is required depends on the scope. In addition to designers, having a dedicated project manager available to help with planning and communication is also helpful.
+
+## #4 New project
+
+This is an opportunity to adopt a good design process from the start, beginning with researching product concepts, user research, etc. New projects likely also have other foundations to lay, so designers need to be flexible.
+
+Similar to redesigns, a team of designers (and a project manager) is ideal to tackle new projects. Once the design foundation has been created, that need will shrink.
+
+## #5 Find a designer
+
+Some projects want to expand their team by bringing in dedicated designers. As designers are often individual contributors with a unique set of interests, experience and availability, this can often be a longer, moree gradual proces.
+
+Simplified designer life cycle in a project:
+
+1. Observer
+	- Sees if there are opportunities to contribute
+	- Decides whether contributing is “worth it”
+	- Ideal if there’s an identifiable go-to person to help get started
+1. New contributor
+	- Has identified an area of improvement
+	- Needs guidance on how to contribute (tools, processes...)
+	- Should feel welcome and get satisfaction from contributing
+1. Occasional contributor
+	- Helps when issues arise
+1. Dedicated contributor
+	- Initiates design improvements
+1. Core contributor
+	- Helps shape overall design and design process in a project
+
+Our goal is clarify what type of design support the project needs, create visibility for the "opening", and help onboard designers.
```

Comment:
The break down in this section is really good, I think it will give a designer a clear indication on how to best fit in depending on the project